cub/cub/device/dispatch/dispatch_histogram.cuh

```diff
@@ -564,6 +564,8 @@ struct DispatchHistogram : SelectedPolicy
 {
   static_assert(NUM_CHANNELS <= 4, "Histograms only support up to 4 channels");
   static_assert(NUM_ACTIVE_CHANNELS <= 4, "Histograms only support up to 4 active channels");
```

Comment:
suggestion: I believe this one can be removed now. This check will be satisfied transitively.

Reply:
Totally! Applied!